db/schema.rb

```diff
@@ -113,4 +113,21 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "issue_updates", force: :cascade do |t|
```

Comment:
really silly question - by creating this and running database migrate you get this file (db/migrate/20230817151230_create_issues.rb)???